seatunnel-e2e/seatunnel-connector-v2-e2e/connector-assert-e2e/src/test/java/org/apache/seatunnel/e2e/connector/assertion/FakeSourceToAssertIT.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.e2e.connector.assertion;
+
+import org.apache.seatunnel.e2e.common.TestSuiteBase;
+import org.apache.seatunnel.e2e.common.container.TestContainer;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.TestTemplate;
+import org.testcontainers.containers.Container;
+
+import java.io.IOException;
+
+public class FakeSourceToAssertIT extends TestSuiteBase {
```

Comment:
This is too simple, add a complex user case? e.g. redis

Reply:
I'll add complex case in later PR.

Reply:
In fact, there is only a small difference between the complex case and the current `connector-jdbc-flink-e2e` module, only the developer needs to pay attention to the data after one run of the test method may affect the next run.
The sink connector needs to be cleaned every time data or generate a specific location from `TestContainer#identifier()`.

Reply:
> In fact, there is only a small difference between the complex case and the current `connector-jdbc-flink-e2e` module, only the developer needs to pay attention to the data after one run of the test method may affect the next run. The sink connector needs to be cleaned every time data or generate a specific location from `TestContainer#identifier()`.

I mean `connector-jdbc-flink-e2e` is not using `@TestTemplate`